prime-router/src/main/kotlin/azure/LookupTableFunctions.kt

```diff
@@ -153,6 +153,7 @@ class LookupTableFunctions(
         return getOktaAuthenticator(PrincipalLevel.SYSTEM_ADMIN).checkAccess(request) { oktaAuthenticatedClaim ->
             val inputData: List<Map<String, String>>
             try {
+                val tableSha256 = request.queryParameters[tableSha256]
```

Comment:
Consider computing this in business logic, `createTable`. Otherwise the burden is on the API user to compute it exactly as expected.

Reply:
So your suggestion is to compute it when the table arrives at the API endpoint.  Note we would have to compute it before creation to check for updates. Not sure I have a preference here, just making the checksum code reusable.

Reply:
Right I'd do it as close to the storage logic as possible to reduce risk. In fact if a `unique` flag is put on the column storing the hash you _could_ just try and save it, it would error if it was already there... I'm not saying it's the right way to do it.

Reply:
Done.  I moved the SHA-256 computing in business logic, createTable.  Thanks.